From 2588d1e6167212f1271d1796a8ea2cea41df1fa2 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Thu, 14 May 2020 08:19:15 +0200
Subject: [PATCH 1/1] Remove Plasma requirements - we are not going to use the
 network setup of the GUI   and it requires way to much overhead, such as
 wayland   and Desktop stack. We are just going to run QT straight   onto the
 framebuffer and use system tools and enclosure   code to handle the hardware
 configuration things.

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 78bfae8..998e239 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -61,7 +61,6 @@ find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS I18n)
 
 if(NOT CMAKE_SYSTEM_NAME STREQUAL "Android")
     find_package(Qt5Widgets ${QT_MIN_VERSION} REQUIRED)
-    find_package(KF5Plasma ${KF5_MIN_VERSION} REQUIRED)
     find_package(KF5DBusAddons ${KF5_MIN_VERSION} REQUIRED)
     find_package(KF5KIO ${KF5_MIN_VERSION} REQUIRED) # FIXME look for "KIOWidgets" (KRun) explicitly
 endif()
-- 
2.20.1

